dask_ml/model_selection/_incremental.py

```diff
@@ -836,9 +851,32 @@ def _get_params(self):
         if self.n_initial_parameters == "grid":
             return ParameterGrid(self.parameters)
         else:
-            return ParameterSampler(self.parameters, self.n_initial_parameters)
+            return ParameterSampler(
+                self.parameters,
+                self.n_initial_parameters,
+                random_state=self.random_state,
```

Comment:
This change is independent of this PR, right?

Reply:
Yes, but I discovered while writing this PR. I ran into a bug while running simulations in December where two versions of Hyperband with the same `random_state` weren't sampling the same parameters. This resolved that.

Should I submit a separate PR for this change?

Reply:
Doesn't matter. Just a single line change (plus tests?) so it's not really complicating the review. But it's nice to get bug fixes in sooner.

Reply:
I submitted https://github.com/dask/dask-ml/pull/496, which also implements a test for `IncrementalSearchCV` (otherwise I wouldn't have submitted it).

I do test this line in this PR (but to make sure that it resolves a different bug, one I ran into in December)